pkg/variantregistry/ocp.go

```diff
@@ -300,7 +300,7 @@ var (
 	s390xRegex              = regexp.MustCompile(`(?i)-s390x|-multi-z-z`)
 	sdnRegex                = regexp.MustCompile(`(?i)-sdn`)
 	serialRegex             = regexp.MustCompile(`(?i)-serial`)
-	singleNodeRegex         = regexp.MustCompile(`(?i)-single-node`)
+	singleNodeRegex         = regexp.MustCompile(`(?i)-single-node|-sno`)
```

Comment:
Might want to make that -sno- so we are sure it isn't a substring of something else.  Also adding some [tests](https://github.com/openshift/sippy/blob/bdfb262d429af922d344d7baa50100beaaa03ac1/pkg/variantregistry/ocp_test.go)  validating some of the job names would be good.